addons/kanban_tasks/view/board/board.gd

```diff
@@ -37,18 +42,19 @@ func _ready():
 
 	notification(NOTIFICATION_THEME_CHANGED)
 
-	await get_tree().create_timer(0.0).timeout
+	button_documentation.pressed.connect(func(): show_documentation.emit())
+	button_documentation.visible = Engine.is_editor_hint()
+
+	button_settings.pressed.connect(settings.popup_centered_ratio_no_fullscreen)
+
+
+func _enter_tree():
 	var ctx: __EditContext = __Singletons.instance_of(__EditContext, self)
 
 	ctx.settings.changed.connect(update)
 
 	ctx.filter_changed.connect(__on_filter_changed_external)
```

Comment:
I have not yet put your changes to test in Godot but could you explain why `__Singletons.instance_of` works reliably in `_enter_tree`. The reason for `await get_tree().create_timer(0.0).timeout` is that the root node is busy setting up children during the `_ready` so it's basically a way of calling the rest of the `_ready` function defferred. From what I can tell the tree root should also be busy during `_enter_tree`.

Reply:
Quick addition before I leave. In other's code instead of await for a timer, I saw await for an explicit update frame. Also when I dealt with the linked issue, the only place when I was able to access everything, was the update function, so I made that workaround. In all other function the tree was not ready or in progress.

Reply:
The errors only occur when a scene from the plugin is opened for edditing. If you close all scenes or install the plugin in a project, like an end user would do, no errors should happen. When you open the scenes the toolscript executes, but the data is not initialized, which causes some issues. I thought about checks for this before but I did not go for it in the end as it does not effect users. But I should propably add a note about it in the README.